pkg/cluster/cluster.go

```diff
@@ -410,11 +414,36 @@ func (c *Cluster) delete() {
 	if c.spec.Backup != nil {
 		k8sutil.DeleteBackupReplicaSetAndService(c.kclient, c.name, c.namespace, c.spec.Backup.CleanupOnClusterDelete)
 	}
+
+	err = c.deleteClientServiceLB()
+	if err != nil {
+		// todo: do not panic!
+		panic(err)
+	}
+}
+
+func (c *Cluster) createClientServiceLB() error {
```

Comment:
We need to capture that this service do roundrobin for N client endpoints. Do you have a better name for this? We already create Client service for each member today.